src/main/java/edu/harvard/iq/dataverse/license/dto/LicenseReorderDto.java

```diff
@@ -0,0 +1,38 @@
+package edu.harvard.iq.dataverse.license.dto;
+
+/**
+ * Helper class used for reordering page.
+ */
+public class LicenseReorderDto {
```

Comment:
The class itself is not specific to reorder functionality - it just happens that it is used in reorder.
I think that some other name should be chosen. Maybe `LicenseSimpleDto` or `LicenseBasicDto`?

Reply:
Then maybe I should just use Tuple with name licenseIdAndLocalizedText which would be even more descriptive.

Reply:
I think that separate class would be easier to implement further changes if it happens (maybe some field can be added or something like that). While using tuple that kind of change would cause to switching it to separate class again.